Sources/SwiftBuildSupport/PackagePIFBuilder+Helpers.swift

```diff
@@ -968,6 +968,7 @@ extension ProjectModel.BuildSettings.Platform {
         case .windows: .windows
         case .wasi: .wasi
         case .openbsd: .openbsd
+        case .freebsd: .freebsd
         default: preconditionFailure("Unexpected platform: \(platform.name)")
```

Comment:
FYI, I discussed this offline with @jakepetroules and will add proper error handling here.

No more `preconditionFailure` shenanigans, as this was brining down Xcode 🫠